tools/js-test-runner/jest-config/jest.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+let config = {};
+
+const defaultConfig = {
+	preset: '@wordpress/jest-preset-default',
+	rootDir: process.cwd(),
+	testEnvironment: 'jsdom',
+	// TODO: remove as soon gutenberg release new version
+	//https://github.com/WordPress/gutenberg/blob/%40wordpress/jest-preset-default%407.1.5-next.33ec3857e2.0/packages/jest-preset-default/jest-preset.js
+	testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/test/*.[jt]s?(x)', '**/?(*.)test.[jt]s?(x)' ],
+	moduleNameMapper: {
+		'\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': path.join(
+			__dirname,
+			'./fileMock.js'
+		),
+		'\\.module\\.(css|less)$': 'identity-obj-proxy',
+		jetpackConfig: path.join( __dirname, './jetpack.config' ),
+	},
+	coverageReporters: [ 'clover', 'lcov' ],
+	collectCoverageFrom: [
+		'**/*.{js,jsx}',
+		'!**/node_modules/**',
+		'!**/vendor/**',
+		'!**/__tests__/**',
+		'!**/coverage/**',
+		'!test-main.{js,jsx}',
+		'!jest.setup.{js,jsx}',
+	],
+	transform: {
+		'\\.[jt]sx?$': [
+			require.resolve( 'babel-jest' ),
+			{ configFile: path.join( __dirname, './babel.config.js' ) },
```

Comment:
I have to wonder if this is the right thing, versus trying the rootDir babel config if that exists.

Reply:
@anomiex this way we need to have `babel.config.js` into each project, right? this is the common pattern?

Reply:
If we have the `babel.config.js` in the project, it seems like it should probably be used. If we don't have one in the project, using a default would be ok.